theme/overrides/changelog.html

```diff
@@ -0,0 +1,66 @@
+{% extends "base.html" %}
+
+{% block site_nav %}
+{% endblock %}
+
+{% block content %}
+
+<h1>Changelog</h1>
+
+<p>
+Follow SEKOIA.IO on <a href="https://twitter.com/sekoia_io">Twitter <span class="twemoji" style="color: #1DA1F2;">{% include ".icons/fontawesome/brands/twitter.svg" %}</span></a> to stay updated on everything we ship.
+</p>
+
+<container class="changelog">
+  
+  <article class="row">
+    <div class="column-left">
+      <span class="alt-mono-font">July 8, 2021</span>
+    </div>
+
+    <div class="column-right">
+      <a href="/changelog/user-center-invitations">
+	New invitation feature in the User Center
+      </a>
+    </div>
+  </article>
+
+  <article class="row">
+    <div class="column-left">
+      <span class="alt-mono-font">July 5, 2021</span>
+    </div>
+
+    <div class="column-right">
+      <a href="/changelog/sekoia-io-new-visual-identity">
+	SEKOIA.IO's new visual identity
+      </a>
+    </div>
+  </article>
+
+  <article class="row">
+    <div class="column-left">
+      <span class="alt-mono-font">July 2, 2021</span>
+    </div>
+
+    <div class="column-right">
+      <a href="/changelog/new-intelligence-souce-hatching-triage">
+	New intelligence source: Hatching Triage
+      </a>
+    </div>
+  </article>
+  
+  <article class="row">
+    <div class="column-left">
+      <span class="alt-mono-font">June 28, 2021</span>
+    </div>
+
+    <div class="column-right">
+      <a href="/changelog/tracking-chinese-apt-groups">
+	Tracking of Chinese APT groups
+      </a>
+    </div>
+  </article>
+    
+</container>
```

Comment:
Can’t we make this section generated? We should probably write a small MkDocs plugin to do so.

Reply:
Having it automatically based on the title and the date in the changelog files would be nice indeed.